recipes/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "spyder-unittest" %}
+{% set version = "0.1b2" %}
```

Comment:
This is the first release of this package, and the idea is to allow people to easily test it through conda-forge ;-)